src/app/tracing/dialog/station-properties/station-properties.component.html

```diff
@@ -1,4 +1,17 @@
-<div mat-dialog-content>
+<div
+    mat-dialog-content
+    cdkDrag
+    cdkDragRootElement=".cdk-overlay-pane"
+    cdkDragBoundary=".cdk-overlay-container"
+>
+    <div class="example-handle">
+        <svg cdkDragHandle width="24px" fill="currentColor" viewBox="0 0 24 24">
```

Comment:
If possible, I'd love a on hover tooltip that tells me, that I can move the dialogue. But that's just my two cents. :)

I know, we're not optimizing for A11y. But ideally, it should also have a screen reader friendly hidden label. But since the app is not really usable with a screen reader, it's not prio.

Reply:
I think, I'll go with Marco's suggestion and simply use a material icon, as we've done in other places in the app. That will fix the reusability.

I'm debating between using [open with](https://materialui.co/icon/open-with), since that's the look I'm going for, or [drag handle](https://materialui.co/icon/drag-handle) because that's semantically the correct icon. 
Though [Angular drag and drop](https://material.angular.io/cdk/drag-drop/overview#customizing-the-drag-area-using-a-handle) is where I got the SVG used here from, and that looks like the open with icon, so we wouldn't really be breaking convention.

On the tooltip, I did set the CSS property `cursor: move`, so it should change your cursor on hover to inform you that this element can be moved. What kind of tooltip where you thinking about?

Lastly, I am unsure whether the move handle placed in the header bar like this is a good look. I'm wondering if I should position the drag handle in line with the first card, or whether we could style the station property dialogue differently, as I think the current look is not very attractive.

Reply:
Checked in with PO, using open_with icon and color black.

Reply:
On open / drag: As long as there's no aria/meta info in the icon, that will convey the wrong message, I'd say pick what ever looks better in this context.

Reply:
On cursor/tooltip: you're right - movable cursor is prolly sufficient